examples/jsm/exporters/GLTFExporter.js

```diff
@@ -1101,24 +1099,35 @@ class GLTFWriter {
 
 				ctx.putImageData( new ImageData( data, image.width, image.height ), 0, 0 );
 
+			} else {
+
+				ctx.drawImage( image, 0, 0, canvas.width, canvas.height );
+
 			}
 
 			if ( options.binary === true ) {
 
-				pending.push( new Promise( function ( resolve ) {
+				let toBlobPromise;
+
+				if ( canvas.toBlob !== undefined ) {
+
+					toBlobPromise = new Promise( ( resolve ) => canvas.toBlob( resolve, mimeType ) );
 
-					canvas.toBlob( function ( blob ) {
+				} else {
+
+					toBlobPromise = canvas.convertToBlob( { type: mimeType } );
```

Comment:
That's a fun switch... 😅

Reply:
Yeah, you'd think they would keep the same method name for OffscreenCanvas?